spec/models/embed/purl_json_loader_spec.rb

```diff
@@ -5,34 +5,56 @@
 RSpec.describe Embed::PurlJsonLoader do
   include PurlFixtures
 
-  subject(:data) { described_class.load('12345') }
+  subject(:loader) { described_class.new('12345') }
+
+  let(:data) { loader.load }
```

Comment:
The result of load is the subject of these tests. So can you please keep this the `subject`?  If you want to test loader.etag, make a different `description` block.  and set the subject to `loader.etag`

Reply:
Fixed.